alibi_detect/saving/saving.py

```diff
@@ -409,14 +448,13 @@ def _save_tokenizer_config(tokenizer: PreTrainedTokenizerBase,
     cfg_token = {}
     logger.info('Saving tokenizer to {}.'.format(filepath))
     tokenizer.save_pretrained(filepath)
-    cfg_token.update({'src': path.joinpath('tokenizer')})
+    cfg_token.update({'src': path.joinpath('tokenizer').as_posix()})
```

Comment:
This is a Windows-specific issue. Basically if we write Windows paths to the `config.toml` (i.e. with `\` instead of `/`), it causes all sorts of pain upon loading.

There are only a few limited places where its neccesary:
- `cfg_model = {'src': local_path.joinpath('model').as_posix()}` in `tensorflow/_saving.py`.
- `cfg_embed.update({'src': local_path.as_posix()})` in `tensorflow/_saving.py`.
- `cfg_token.update({'src': path.joinpath('tokenizer').as_posix()})` in `saving.py`.

It isn't required for some other operations such as:

```python
save_path = filepath.joinpath(X_REF_FILENAME)
```

because currently `X_REF_FILENAME = 'x_ref.npy'`. (if we did e.g. `data/x_ref.npy` it'd need updating with `as_posix()`. 
 
I've just made a commit with `as_posix()` moved to `saving._path2str` so that any necessary paths are automatically converted, instead of us having to think about whether to do each in-place. Will see if it passes on the Windows build...